app/models/broadcast_event/order_made.rb

```diff
@@ -7,6 +7,35 @@ class BroadcastEvent::OrderMade < BroadcastEvent
 
   # The seed itself can be a subject
   seed_subject :order
+
+  # !CAUTION!
+  # Be careful before changing this behaviour, as it could impact
+  # on users of the events warehouse.
+  # Discussion with jgrg 29/10/2020:
+  # > jgrg: I guess the reason removing "event_types.key = 'order_made'" pulls in
+  # > samples from other projects is that some events involve samples from
+  # > multiple projects, such as when samples from multiple projects are
+  # > loaded on the same flowcell. Will "order_made" always be confined to
+  # > samples in our project
+  #
+  # This assumption is currently *true* for all order_made events, however it
+  # will be potentially impacted if we change the behaviour of cross-study
+  # or cross-project orders.
+  #
+  # A cross-study/cross-project order is currently generated if sequencing
+  # is requested *directly* on multiplexed library containing samples from
+  # different studies/projects. This mostly occurs with re-sequencing, but
+  # can also happen during GBS, custom-pooling, or situations in which the
+  # sequencing request creation is deferred until after multiplexing.
+  #
+  # Currently these orders are not explicitly associated with *any* study/project
+  # (ie. study_id and project_id on order are nil), and this event uses this
+  # explicit link. As a result, these orders have no projects/studies associated
+  # as subjects.
```

Comment:
Had to read this paragraph a few times. If this is still correct, suggest changing to:
```
# Currently these orders are not explicitly associated with *any* study/project
# (ie. study_id and project_id on order are nil)
# The order_made event links projects/studies as subjects only if this explicit link is present.
# As a result, these orders have no projects/studies associated as subjects.
```